cmake/sycl.cmake

```
@@ -0,0 +1,33 @@
+# IntelSYCL for dpcpp and icpx if the config is existed and cmake reaches the requirement
+if(CMAKE_CXX_COMPILER MATCHES "dpcpp|icpx")
+    if(CMAKE_HOST_WIN32 AND CMAKE_VERSION VERSION_GREATER_EQUAL 3.25)
+        find_package(IntelSYCL QUIET)
```

should this be optional?
```suggestion
        find_package(IntelSYCL QUIET REQUIRED)
```

---

yes it is optional. If there's no IntelSYCL, we have -fsycl for that.